lib/senders/templates/new_device_login.txt

```diff
@@ -5,6 +5,8 @@
 {{#if ip}}{{t "IP address: %(ip)s" }}{{/if}}
 {{#if timestamp}}{{ timestamp }}{{/if}}
 
+{{t "Manage account here  %(link)s"}}
```

Comment:
Lets update this to match how the other text links are, https://github.com/mozilla/fxa-auth-server/blob/f3261a613780cdec738069d262bc759c046041a3/lib/senders/templates/post_change_primary.txt#L5